x-pack/plugins/maps/public/shared/layers/sources/ems_file_source/ems_file_source.js

```diff
@@ -38,7 +43,12 @@ export class EMSFileSource extends AbstractVectorSource {
     const emsFiles = await getEmsVectorFilesMeta();
     const meta = emsFiles.find((source => source.id === this._descriptor.id));
     if (!meta) {
-      throw new Error(`Unable to find EMS vector shapes for id: ${this._descriptor.id}`);
+      throw new Error(i18n.translate('xpack.maps.source.emsFile.errorMessage', {
```

Comment:
`errorMessage` id is very vague. How about `xpack.maps.source.emsFile.unableToFindIdErrorMessage`?